lib/model/model.dart

```diff
@@ -0,0 +1,289 @@
+import 'package:we_find/extensions/xml_extensions.dart';
+import 'package:we_find/model/I18NString.dart';
+import 'package:xml/xml.dart';
+
+/// Representation of a 'course' tag
+/// that appears in the XML response of the u:find API
+/// 'course' tags are direct descendants of the 'result' root-tag.
+class Course {
+  final String? id;
```

Comment:
Why isn't this I18NString?

Reply:
Because of the structure of the XML response. Only tags with an attribute "xml:lang" are represented with `I18NString`. Furthermore, the `id` of the course is a string of 6 digits with leading zeros, therefore it would not make sense at all to internationalize it.

A course tag has the following attributes:

`<course id="050000" when="2021S" version="2021-04-21T11:25:54+02:00">`

You might want to take a look at a full example response, such as [this one](https://m-ufind.univie.ac.at/courses/?query=spl5 count:1).

Reply:
got it, squash and merge then! :+1: